examples/js/loaders/MMDLoader.js

```diff
@@ -427,7 +427,7 @@ THREE.MMDLoader.prototype.pourVmdIntoCamera = function ( camera, vmd, name ) {
 
 		var clip = new THREE.AnimationClip( name === undefined ? THREE.Math.generateUUID() : name, -1, tracks );
 
-		if ( clip !== null ) {
```

Comment:
Maybe no.... I don't remember why I added this null check logic....
Seems like I should remove.